runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/PTransformTranslation.java

```diff
@@ -63,6 +64,8 @@
   public static final String GROUP_BY_KEY_TRANSFORM_URN =
       getUrn(StandardPTransforms.Primitives.GROUP_BY_KEY);
   public static final String IMPULSE_TRANSFORM_URN = getUrn(StandardPTransforms.Primitives.IMPULSE);
+  public static final String STREAMING_IMPULSE_TRANSFORM_URL = "flink:transform:streaming_impulse:v1";
```

Comment:
It isn't necessary to modify this class. The URL is Flink runner specific and should live with the Flink runner.

Reply:
👍 moved to FlinkStreamingPortablePipelineTranslator